3.evaluate_model/evaluate_model.sh

```diff
@@ -7,3 +7,7 @@ jupyter nbconvert --to python \
 jupyter nbconvert --to python \
         --FilesWriter.build_directory=scripts/nbconverted \
         --execute class_PR_curves.ipynb
+
+jupyter nbconvert --to python \
+        --FilesWriter.build_directory=scripts/nbconverted \
+        --execute confusion_matrices.ipynb
```

Comment:
I believe Greg mentioned we should try to avoid using wildcards with `nbconvert`, the logic being that we don't want to run these any newly added notebooks on accident. Once I finish with the module though, I will confirm with Greg that it is best to use individual `nbconvert` commands instead of a wildcard.